rts/lra/test/basic/src/test/java/io/narayana/lra/arquillian/NonRootLRAParticipantIT.java

```diff
@@ -76,8 +79,9 @@ public void testNonRootLRAParticipantEnlist() {
      * @throws URISyntaxException if arquillian URL is invalid
      */
     @Test
-    public void testFinishLRA() throws URISyntaxException {
-        NarayanaLRAClient client = new NarayanaLRAClient(); // the narayana client API for using LRAs
+    public void testFinishLRA() throws URISyntaxException, MalformedURLException {
```

Comment:
Remark: adding an extra exception is fine from my perspective (so I am not asking for a change) but, in the past, you have told me not to do that in my own PRs.

Reply:
No, you complained about the exact same thing you have done here.